README.md

```diff
@@ -88,7 +88,7 @@ clip_inference turn a set of text+image into clip embeddings
 * **write_batch_size** Write batch size (default *10**6*)
 * **wds_image_key** Key to use for images in webdataset. (default *jpg*)
 * **wds_caption_key** Key to use for captions in webdataset. (default *txt*)
-* **clip_model** CLIP model to load (default *ViT-B/32*)
+* **clip_model** CLIP model to load (default *ViT-B/32*). Specifiy it as `"open_clip:Vit-B/32-quickgelu"` to use the [open_clip](https://github.com/mlfoundations/open_clip) models.
```

Comment:
Can you also say here that to use open clip, it's needed to `pip install open_clip_torch` ?

We can also consider adding it directly in the dependencies if it doesn't transitively depend on too many things

Reply:
I'm not sure what requirements it has but the environment.yml in its repo is quite big.

Reply:
The actual requirements of the pypi package are listed there https://github.com/mlfoundations/open_clip/blob/main/setup.py#L48

Reply:
It seems to me that means 0 new transitive dependencies
Worth checking by installing it in an environment where clip retrieval is already installed

Reply:
i think they have conflicting dependencies for webdataset as specified but it seems to work anyway